vector/src/main/res/drawable/ic_fab_add_by_qr_code.xml

```diff
@@ -0,0 +1,30 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="37dp"
+    android:height="36dp"
+    android:viewportWidth="37"
+    android:viewportHeight="36">
+  <path
```

Comment:
This is identical with `ic_fab_add_by_mxid.xml`. I do not know where the icon comes from. Maybe I can ask to our design team to provide an alternative icon for qr code

Reply:
See [above](https://github.com/vector-im/element-android/pull/2289#discussion_r518279698)